src/common/components/search-bar/index.tsx

```diff
@@ -123,12 +119,17 @@ const SearchBar: React.FC<SearchBarProperties> = ({
 							</Button>
 						</div>
 					) : (
-						<IconButton className={styles["search__icon_button"]}>
```

Comment:
I offer another option, just add optional props, like i did? but not pushet yet
![image](https://github.com/user-attachments/assets/ac5ac1da-d5eb-495b-ab46-a8b92865feca)

Reply:
which props do you mean?